README.md

```diff
@@ -78,12 +77,61 @@ An event type (e.g. `CI`, `CD`, `TestExecution`) corresponds to the step of your
     --test_suite "<test_suite>" \
     --test_stats "success=5,failure=0,total=5" \
     --test_start_time "1626804356000" \
-    --test_end_time "Now"
+    --test_end_time "Now" \
+    --full
```

Comment:
Note: This is the only place where full is currently required. I even left it off of the CI and CD events event when the whole event is provided all at once.